README.md

```diff
@@ -15,19 +15,21 @@ Simple shortened example for listing all pipelines:
 ```go
 import (
     "github.com/buildkite/go-buildkite/v3/buildkite"
+    "gopkg.in/alecthomas/kingpin.v2"
 )
-...
 
-config, err := buildkite.NewTokenConfig(*apiToken, false)
+var (
+    apiToken = kingpin.Flag("token", "API token").Required().String()
+    org = kingpin.Flag("org", "Organization slug").Required().String()
+)
+
+client, err := buildkite.NewOpts(buildkite.WithTokenAuth(*apiToken))
 
 if err != nil {
-	log.Fatalf("client config failed: %s", err)
+	log.Fatalf("New client configuration failed: %s", err)
```

Comment:
It's likely that this would be wrapped, so best to keep it lower case.

I also don't think we need "new".

Reply:
Reverted back 👍